### ISSUE TITLE
Typo in readme

### ISSUE THREAD

**Issue by michalrudko** [ksauzz/krbticket#32]
```
retry_options = {
  'wait_exponential_multiplier': 1000,
  'wait_exponential_multiplier': 10000,
  'stop_max_attempt_number': 5
}
```

should be

```
retry_options = {
  'wait_exponential_multiplier': 1000,
  'wait_exponential_max': 10000,
  'stop_max_attempt_number': 5
}
```

right?

Plus - could you please provide some description what these parameters actually mean? Thanks!

**Comment by ksauzz** [comment]
> should be
> 
>```
>retry_options = {
>  'wait_exponential_multiplier': 1000,
>  'wait_exponential_max': 10000,
>  'stop_max_attempt_number': 5
>}
>```
>
>right?

Right. That's typo. I'll fix it. :smiley: 

>Plus - could you please provide some description what these parameters actually mean? Thanks!

A retry happens when `kinit` fails because of network issue, or something. 
These parameters are used for [exponential backoff sleep](https://github.com/rholder/retrying/blob/v1.3.3/retrying.py#L171-L178) for retrying kinit. 
A sleep time is `x^2 * wait_exponential_multiplier(ms)` between each retry, upto wait_exponential_max (ms), then wait_exponential_max (ms) afterwards. Retrying stops if attempt count is over `stop_max_attempt_number`. If you'd like to use other sleep behavior you can choose your preferable one from [here](https://github.com/rholder/retrying/tree/v1.3.3#examples).
`retrying_options` is just passed to `@retry` decorator of [retrying](https://github.com/rholder/retrying/tree/v1.3.3#retrying).

**Comment by ksauzz** [comment]
Fixed by #33
